### DISCUSSION TITLE
v11 crashing with smart light bulb

### DISCUSSION THREAD

**Discussion by ralphrmartin** [arendst/Tasmota#14939] [Q&A] [answered]
I have a smart bulb which worked perfectly with v9 and v10. Installing Tasmota v11, the bulb initially shows a web interface, but loses connectivity some time afterwards. Here is the device info as shown.
Let me know if there is more I can do to help to find this issue. 

`<div style="padding: 5px; font-size: 16px; color: rgb(234, 234, 234); font-family: verdana, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(37, 37, 37); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; text-align: center;"><h3>LB01-15W-E27-TAS</h3><h2>RalphsNewBulb</h2></div><div id="i" name="i" style="padding: 5px; font-size: 16px; color: rgb(234, 234, 234); font-family: verdana, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(37, 37, 37); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Program Version | 11.0.0(tasmota)
-- | --
2022-02-12T14:13:50
2_7_4_9/2.2.2-dev(38a443e)
0T00:00:16
959 at 0xFA000
36
Software/System restart
RalphsNewBulb
 
rrmair (60%, -70 dBm) 11n
tasmota-D3F022-4130
C4:4F:33:D3:F0:22
192.168.1.238
192.168.1.254
255.255.255.0
192.168.1.102
1.1.1.1
 
Enabled
 
Disabled
 
None
 
13889570 (ESP8266EX)
0x1540A1
2048 kB
1024 kB
620 kB
380 kB
26.8 kB

</div>`

**Comment by ralphrmartin** [comment]
Hmm, 11.0.0.2 seems stable, although 11.0.0 is not - 11.0.0.2  has now been up for over 5 hours. I will restart 11.0.0.2 and see if it stays stable.

I tried 11.0.0 and got crashes, reverted to 10 and it was stable, reinstalled 11.0.0 and crashes started again, so I do not believe this is a hardware issue; furthermore v 9 and v 10 were running stably on this bulb for many months.

**Reply by Jason2866** [reply]
Solution use 11.0.0.2 😏 If! there is a issue in 11.0.0 the fix will be done in development and is included in the next release. No updates done to released versions.
Looking at the commits since 11.0.0 there is no relevant code change for ESP8266 driven devices.

**Comment by ralphrmartin** [comment, marked as answer]
Sure, but it would be useful to understand _why_ 11.0.0.2 fixes the issue, so it doesn't come back later. 😏 
Anyway, I will keep monitoring 11.0.0.2, and let you know if the issue comes back.

**Reply by Jason2866** [reply]
Since we cant reproduce any issue with 11.0.0 and PWM driven devices we cant search anything.

**Reply by ralphrmartin** [reply]
Fair enough, if no one else is affected.

**Reply by stefanbode** [reply]
Same problem and solution with my pWM led stripes. 11.0 devices just dies after some time. Current dev s fine. No problems anymore

**Comment by wegotourselvesareader** [comment]
Came here to report this very issue, and found an existing issue and discussion already in place.

Looks like I've a different bulb to the OP; mine is this: https://templates.blakadder.com/nedis_WIFILW10WTB22.html

Sometimes the bulb drops off the network within a couple of minutes, sometimes it stays online for several hours before dropping off; I haven't been able to identify a common pattern yet.

Willing to give 11.0.0.2 a go, but am struggling to find a compiled binary. Will I need to get to grips with compiling my own? Don't mind doing so, but it'll have to wait until I get some spare time at the weekend.

**Reply by Jason2866** [reply]
You find precompiled firmware from latest development version here https://github.com/arendst/Tasmota-firmware

**Reply by wegotourselvesareader** [reply]
> You find precompiled firmware from latest development version here https://github.com/arendst/Tasmota-firmware

That's fantastic, thank you.

**Reply by wegotourselvesareader** [reply]
Two days now on 11.0.0.2, and the bulb has been stable. Seems good so far.
